pyproject.toml

```
@@ -16,7 +16,7 @@ classifiers = [
 Changelog = "https://github.com/ginger-io/transformer-embeddings/releases"
 
 [tool.poetry.dependencies]
-python = "^3.7,<3.11"
+python = ">=3.8,<3.11"
```

Nit: I prefer the `^` definitions more. [Docs.](https://python-poetry.org/docs/dependency-specification/)

```suggestion
python = "^3.8,<3.11"
```